tests/scripts/run-e2e-tests.js

```diff
@@ -191,6 +198,8 @@ yargs
           return acc;
         }, []);
 
+        const debugFlag = debug ? ['--debug'] : [];
```

Comment:
```suggestion
        const optionFlags = [];
        if(debug) {
          optionFlags.push('--debug');
        }
```
Something like that would be more future-proof if we have any other options to add, but I'm also fine with it how you have it